src/ucs/async/eventfd.c

```diff
@@ -0,0 +1,79 @@
+/**
+* Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2023-2024. ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#include "eventfd.h"
+
+#include <ucs/debug/log.h>
+#include <ucs/sys/sys.h>
+
+
+ucs_status_t ucs_async_eventfd_create(int *fd)
+{
+
+    *fd = eventfd(0, EFD_CLOEXEC | EFD_NONBLOCK);
```

Comment:
1. save to local var until we know it's success
2. use UCS_ASYNC_EVENTFD_INVALID_FD